tv/components/RoomDisplay.tsx

```diff
@@ -27,7 +35,15 @@ const RoomDisplay = ({ room, currentSession, nextSession }: RoomProps): JSX.Elem
 
             <div className="checkin-qrcode">
```

Comment:
`checkin-qrcode` could be renamed to `checkin-container` to avoid confusion on QR code styling.